tests/Facebook/InstantArticles/Elements/InstantArticleTest.php

```diff
@@ -473,4 +473,133 @@ public function testIsRTLEnabled()
 
         $this->assertEquals($expected, $result);
     }
+
+    public function testDeleteChildren()
```

Comment:
Please add a testDeleteOnlyChild() test case, where you add only 1 paragraph and delete it. This will throw that error on an invalid state about `$this->children` array.

This is why we need it to be initialized with an empty array, not a null one.